lib/Newsletter/Renderer/Blocks/Image.php

```diff
@@ -14,11 +14,17 @@ static function render($element, $column_base_width) {
       $element['src'] = get_option('siteurl') . $element['src'];
     }
 
-    $element['width'] = (int)$element['width'];
-    $element['height'] = (int)$element['height'];
-    $element = self::adjustImageDimensions($element, $column_base_width);
+    $width_without_px = str_replace('px', '', $element['width']);
```

Comment:
Please consider adding automated tests for this change.

Reply:
Repushed with a more bulletproof check. Let me know what you think about that. It does the following:
- If with & height are in pixels, it processes them as it always did.
- If width & height are anything else, it just uses them and sets `max-width` to `100%`.

This way even if we get `null` or `''` or `auto` we would get the correct max-width. Moreover, this should not come from StructureTransormer which converts nulls (from Gutenberg) to "auto".